src/chaos-experiments/models/kubernetes/chaosExperimentConnector.js

```diff
@@ -0,0 +1,41 @@
+const util = require('util');
+const fs = require('fs');
+const kubernetesConfig = require('../../../config/kubernetesConfig');
+const logger = require('../../../common/logger');
+const requestSender = require('../../../common/requestSender');
+const kubernetesUrl = kubernetesConfig.kubernetesUrl;
+const kubernetesNamespace = 'apps'; // kubernetesConfig.kubernetesNamespace;
```

Comment:
fix to not be hardcoded but take it from the kubeObject

Reply:
fixed